app/src/main/res/layout/favorite_transfer_account_spinner_item.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please, try to fix the editor warnings if possible. For example, in this line it says the `<LinearLayout>` and its contents can be replaced by a single `<TextView>` with a `drawableEnd` attribute.